libafl_frida/src/alloc.rs

```diff
@@ -556,12 +556,14 @@ impl Allocator {
             &mut |range: &RangeDetails| -> bool {
                 let start = range.memory_range().base_address().0 as usize;
                 let end = start + range.memory_range().size();
-
-                if !self.is_managed(start as *mut c_void) {
+                //the poisoned region should be the highest valid userspace region
```

Comment:
I agree with your patch, but the comment seems wrong. Can you explain what you meant?

Reply:
That should really say shadow region not poisoned region... I'll fix that.